nl/editor/ui/chrome/composer/editorOverlay.dtd

```diff
@@ -605,7 +605,7 @@ entities below were automatically added by l10nmerge
 <!ENTITY filesave.accesskey "s">
 <!ENTITY pasteLinkCmd.label "Link">
 <!ENTITY fonthelvetica.accesskey "l">
-<!ENTITY printButtonCmd.label "Print…">
+<!ENTITY printButtonCmd.label "Afdrukken">
```

Comment:
CORRECTION MIGHT BE NEEDED, it should be: ''Afdrukken...''

- assuming we want to keep the '...' in there.